src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -700,18 +727,20 @@ def _handle_group_actions(
             return self.respond()
 
         response_url = slack_request.data["response_url"]
-        json_blocks = orjson.dumps(response.get("blocks")).decode()
         webhook_client = WebhookClient(response_url)
         try:
             webhook_client.send(
-                blocks=json_blocks,
+                blocks=response.get("blocks"),
```

Comment:
don't need to do this, originally that orjson line only converted from single quotes to double quotes (i am assume this was because in pre-slack_sdk world, we needed to. the slack_sdk handles this for us, i verified this with my local setup